src/controllers/users.js

```diff
@@ -60,11 +60,11 @@ const updateUserById = (req, res, next) => {
 const loginUser = async (req, res, next) => {
   let payload = req.body;
   if (!payload.password || !payload.email) return res.status(400).json({ error: "username and password required" })
-  const userExists = await model.getUserByEmail(payload.email)
-  if (userExists.error) return res.status(404).json({ error: "user does not exist" })
-  const isValid = await bcrypt.compare(payload.password, userExists.password);
+  const user = await model.getUserByEmail(payload.email)
```

Comment:
wasn't this change in another PR? Why is it in here? Is this what you were referencing to on your slack message?

Reply:
I remember, it is because Nick changed the variable `userExists` to `user`